x-pack/plugins/security_solution/public/common/components/event_details/investigation_guide_view.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiHorizontalRule, EuiTitle, EuiText } from '@elastic/eui';
```

Comment:
oh I see now that the investigation guide test was removed because this piece was extracted out (great call!) we could consider bringing the unit test back in a separate file as well.

Reply:
The test is here https://github.com/elastic/kibana/pull/113274/files#diff-77c98831ff0db1540147d9e4f50af84fed199638a23e831d0113e06dc8815745R130